api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -494,6 +494,7 @@ message Features {
   // AccessControls enables FIPS access controls
   bool AccessControls = 6 [(gogoproto.jsontag) = "access_controls"];
   // AdvancedAccessWorkflows enables advanced access workflows
+  // Deprecated: Use AccessRequests field instead.
   bool AdvancedAccessWorkflows = 7 [(gogoproto.jsontag) = "advanced_access_workflows"];
```

Comment:
Btw, could you use the proto deprecation option too?

It's something like this:

```suggestion
  bool AdvancedAccessWorkflows = 7 [(gogoproto.jsontag) = "advanced_access_workflows", deprecated=true];
```